api/routes/index.js

```diff
@@ -2,10 +2,12 @@ const express = require("express");
 
 const movieRouter = require("./movieRouter");
 const userRouter = require("./userRouter");
+const reviewRouter = require("./reviewRouter");
 
 const router = express.Router();
 
 router.use("/movies", movieRouter);
 router.use("/users", userRouter);
+router.use("/review", reviewRouter);
```

Comment:
```suggestion
router.use("/reviews", reviewRouter);
```
index 내의 다른 엔드포인트들과의 통일성을 위해 맞춰주세요.